ios/RNCAsyncStorage.m

```diff
@@ -269,7 +288,17 @@ - (instancetype)init
   if (!(self = [super init])) {
     return nil;
   }
-  RCTStorageDirectoryMigrationCheck();
+
+  // Both directories from our deprecated path (Documents) must be migrated to our new path (Application Support/[bundleID])
+  static dispatch_once_t onceTokenOldStorage;
+  dispatch_once(&onceTokenOldStorage, ^{
```

Comment:
Do we actually need `dispatch_once` here? These seem to only be called more than once if in dev mode.

Reply:
I can delete the dispatch_once calls. The method doesn't do anything if data was already migrated over anyway.  Not sure I know what you mean about dev mode affecting this.

Reply:
I meant during a normal dev session, one would reload the bridge, thereby creating a new instance of this module.